datadog_checks_dev/pyproject.toml

```
@@ -70,6 +70,7 @@ cli = [
     "orjson",
     "packaging",
     "pip-tools",
+    "pathspec",
```

```suggestion
    "pathspec>=0.10.0",
```
The gitignore module is not available in the 0.9.0 release